packages/react-core/src/demos/DataList/examples/DataListPagination.tsx

```diff
@@ -0,0 +1,315 @@
+                    </DataListCell>,
+                    <DataListCell isFilled={false} alignRight key="secondary content align">
+                      <Flex>
+                        <FlexItem>
+                          <Button variant="secondary">Action</Button>
+                        </FlexItem>
+                        <FlexItem>
+                          <a href="#">Link</a>
+                        </FlexItem>
+                      </Flex>
+                    </DataListCell>
+                  ]}
+                />
+              </DataListItemRow>
+            </DataListItem>
+            <DataListItem aria-labelledby="Demo-item5">
+              <DataListItemRow>
+                <DataListItemCells
+                  dataListCells={[
+                    <DataListCell isFilled={false} key="buttons5">
+                      <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                          <Text component={TextVariants.small}>patternfly-elements</Text>
+                        </FlexItem>
+                        <FlexItem>
+                          <Text component={TextVariants.small}>PatternFly elements</Text>
+                        </FlexItem>
+                        <FlexItem>
+                          <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+                            <FlexItem>
+                              <CodeBranchIcon /> 5
+                            </FlexItem>
+                            <FlexItem>
+                              <CodeIcon /> 9
+                            </FlexItem>
+                            <FlexItem>
+                              <CubeIcon /> 2
+                            </FlexItem>
+                            <FlexItem>
+                              <CheckCircleIcon /> 11
+                            </FlexItem>
+                            <FlexItem>
+                              <ExclamationTriangleIcon /> 4
+                            </FlexItem>
+                            <FlexItem>
+                              <TimesCircleIcon /> 1
+                            </FlexItem>
+                            <FlexItem> Updated 2 days ago</FlexItem>
+                          </Flex>
+                        </FlexItem>
+                      </Flex>
+                    </DataListCell>,
+                    <DataListCell isFilled={false} alignRight key="secondary content align">
+                      <Flex>
+                        <FlexItem>
+                          <Button variant="secondary">Action</Button>
+                        </FlexItem>
+                        <FlexItem>
+                          <a href="#">Link</a>
+                        </FlexItem>
+                      </Flex>
+                    </DataListCell>
+                  ]}
+                />
+              </DataListItemRow>
+            </DataListItem>
+            <DataListItem aria-labelledby="pagination">{renderPagination()}</DataListItem>
```

Comment:
should the pagination render in a `DataListItem`?

Reply:
Agreed, it should be rendered outside the DataList, probably just before the closing PageSection tag